### ISSUE TITLE
Grails 3.1.0 not showing Bootstrap glyphicons

### ISSUE THREAD

**Issue by GregorioBelinchonSaiz** [grails/grails-core#9679]
I am using bootstrap on a GRAILS 3.1.0 application with
``````
compile 'org.webjars:bootstrap:3.3.6'
``````
In the index.jsp I set
``````
<span class="glyphicon glyphicon-edit" aria-hidden="true"></span>
``````
In application.js
``````
//= //require jquery-2.1.3.js
//= require /webjars/bootstrap/3.3.6/js/bootstrap.min
//= require_tree .
//= require_self
``````
In application.css
``````
*= require main
*= require mobile
*= require /webjars/bootstrap/3.3.6/css/bootstrap.min
*= require_self
``````

When I run the app both dev and prod from grails ...
``````
grails prod run-app
grails dev run-app
``````
... all work propertly, both bootstap components and glyphicons, but when I put the war on a Tomcat, the glyphicons are not shown, but the bootstrap components works propertly.

I have seen some errors on traces with the form
``````
No mapping found for HTTP request with URI [/testglyphs-0.1/fonts/glyphicons-halflings-regular.ttf] in DispatcherServlet with name 'grailsDispatcherServlet'
``````
The files could be found on Tomcat in the folder

apache-tomcat-8.0.30\webapps\testglyphs-0.1\assets\webjars\bootstrap\3.3.6\fonts

This happends in Grails 3.1.0 and 3.1.1.
I deploy war file on Tomcat 8.0.30 64bits
I attach an example project.
[testglyphs.zip](https://github.com/grails/grails-core/files/126615/testglyphs.zip)

**Comment by GregorioBelinchonSaiz** [comment]
Hi Graeme,
It seems that this issue still occurs in grails-3.1.3
Using the same example testgplyphs, replacing in gradle.properties the grails version to 3.1.3, when I build  the project and put the war on tomcat 8.0.20,I can't see the glyphicon-edit icon, while it can be seen when running with "bootRun" from gradle.

Is it needed to put the bootstrap icons library on any grails repository?

Thanks in advance.

**Comment by GregorioBelinchonSaiz** [comment]
Hi Graeme,
It seems that this issue still occurs in grails-3.1.3
Using the same example testgplyphs, replacing in gradle.properties the grails version to 3.1.3, when I build  the project and put the war on tomcat 8.0.20,I can't see the glyphicon-edit icon, while it can be seen when running with "bootRun" from gradle.

Is it needed to put the bootstrap icons library on any grails repository?

Thanks in advance.

**Comment by graemerocher** [comment]
@GregorioBelinchonSaiz This seems more like a bug in asset pipeline. Does asset pipeline support web jars @davydotcom ?

**Comment by davydotcom** [comment]
It does support web jars but that being said 9 times out of ten they are not properly packaged. Ill take a look and see what i can find

**Comment by davydotcom** [comment]
Located the issue thanks for submitting your sample project. This is fixed in gradle plugin version `2.7.4` and grails asset-pipeline plugin `3.1.4`

**Comment by davydotcom** [comment]
grails plugin `3.1.3` sorry had a typo corrected above
